src/plugins/line-numbers/prism-line-numbers.ts

```diff
@@ -1,3 +1,4 @@
+import Prism from '../../global';
```

Comment:
I would lowercase to not confuse this with the `Prism` class:

```suggestion
import prism from '../../global';
```